### PR TITLE
2020-10-01 SDK model changes for Peering

### PR DESCRIPTION
**Pull request by swetha-anantharajaiah**
Swagger Pr link : https://github.com/Azure/azure-rest-api-specs/pull/11044

### CODE REVIEW

**Comment by bquantump** [comment]
@swetha-anantharajaiah 

Any update here on fixing the CI? If not will close in the next couple days.

**Comment by swetha-anantharajaiah** [comment]
> @swetha-anantharajaiah
> 
> Any update here on fixing the CI? If not will close in the next couple days.

Its fixed

File: sdk/peering/Microsoft.Azure.Management.Peering/src/Generated/IPeeringManagementClient.cs [hunk 1]
```diff
@@ -142,19 +147,17 @@ public partial interface IPeeringManagementClient : System.IDisposable
...
+        /// location and service provider.
         /// </param>
         /// <param name='customHeaders'>
         /// The headers that will be added to request.
         /// </param>
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        Task<AzureOperationResponse<string>> CheckServiceProviderAvailabilityWithHttpMessagesAsync(string peeringServiceLocation = default(string), string peeringServiceProvider = default(string), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<string>> CheckServiceProviderAvailabilityWithHttpMessagesAsync(CheckServiceProviderAvailabilityInput checkServiceProviderAvailabilityInput, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```
Comment (nisha-bhatia) [inline sdk/peering/Microsoft.Azure.Management.Peering/src/Generated/IPeeringManagementClient.cs:160]: This is a breaking change. Please let us know if we're missing something. Thanks.

**Review by nisha-bhatia — CHANGES_REQUESTED** [review]

**Comment by bquantump** [comment]
@swetha-anantharajaiah are you planning on resolving this? If we don't hear anything by the end of the week, we will be closing this.

File: sdk/peering/Microsoft.Azure.Management.Peering/src/Generated/IPeeringManagementClient.cs (hunk 1 above)
Comment (swetha-anantharajaiah) [reply sdk/peering/Microsoft.Azure.Management.Peering/src/Generated/IPeeringManagementClient.cs:160]: There was no change in the rest api spec for this and used the same autorest cmd to generate the SDK as before. Am I missing something here ?

**Comment by swetha-anantharajaiah** [comment]
> @swetha-anantharajaiah are you planning on resolving this? If we don't hear anything by the end of the week, we will be closing this.

Resolved

**Comment by bquantump** [comment]
@swetha-anantharajaiah please fix CI or give us and update else we will be closing at end of the week.

**Comment by nisha-bhatia** [comment]
Overrode check enforcer because pipeline manually ran: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=626589&view=results

**Review by nisha-bhatia — APPROVED** [review]

**Pull request merged**